cmd/apiserver/main.go

```diff
@@ -35,8 +35,9 @@ func main() {
 		// +kubebuilder:scaffold:resource-register
 		WithResource(&clusterv1alpha1.ClusterGateway{}).
 		WithLocalDebugExtension().
-		DisableAuthorization().
+		//DisableAuthorization().
```

Comment:
ping @Somefive , does it block kubevela here?

Reply:
lets remove the line (instead of commenting)

Reply:
I confirmed with @yue9944882 , and remove the code.